pymc/backends/arviz.py

```diff
@@ -165,6 +165,17 @@ def __init__(
                         UserWarning,
                     )
             self.ntune = len(self.trace) - self.ndraws
+
+            for strace in self.trace._straces.values():
+                if strace._stats is not None:
+                    for stat in strace._stats:
```

Comment:
`BaseTrace` does not have a `_stats` attribute !

It's just the `NDArray` that has it.